### ISSUE TITLE
hyperparameter settings

### ISSUE THREAD

**Issue by 1124146862** [postBG/CosMo.pytorch#14]
Hi, I ran the example code (fashioniq dataset) with the default configuration provided in the thesis project, but couldn't achieve the results reported in the thesis. I wonder if there is a problem with my hyperparameter settings (adjusted random seeds, learning rate, etc.), the highest can achieve a top@50 accuracy of 44% on the toptee sub-dataset (the value reported in the paper is at 57% about).
If I want to reproduce the experimental results in the paper, could you please give me some suggestions for my experiments.

**Comment by numpee** [comment]
Hi, 
Please check the "Notes on Evaluation" section in our README.
